.travis.yml

```diff
@@ -10,13 +10,11 @@ node_js: '8'
 install: npm install
 before_script:
   - npx cypress verify &
-  - cd stories
-  - npm install
-  - cd ..
 script:
   - kcd-scripts validate lint,build-and-test,test:cover
   - kcd-scripts validate test:ssr,test:cypress
-  - kcd-scripts validate test:flow:coverage,test:ts
+  # TODO: fix typescript/flow
+  # - kcd-scripts validate test:flow:coverage,test:ts
```

Comment:
Could I get a TypeScript maintainer to give this a look please? It's failing the build: https://travis-ci.org/paypal/downshift/builds/437768573

Reply:
@kentcdodds I did some research, and it turned out that this is a bug introduced by an outdated version of `@types/jquery` that is currently referenced by `cypress/cli` (https://github.com/cypress-io/cypress/issues/2363). They have merged the fix (by bumping up the version of `@types/jquery`, but not yet published that fix yet (https://github.com/cypress-io/cypress/pull/2369). I think in their next release, this fix will be published. However, before that, the quick and dirty fix for us is to do `npm i --save-dev @types/jquery@3.3.6`, or reference `@types/jquery@3.3.6` directly in `package.json`. What do you think?

Reply:
If you think this fix is too hacky, then I'd say just go ahead and publish 3.0.0 without fixing this, until cypress publishes their next release.